client/package.json

```diff
@@ -28,6 +28,7 @@
     "axios": "^0.26.0",
     "date-fns": "^2.28.0",
     "dayjs": "^1.11.1",
+    "lodash": "^4.17.21",
```

Comment:
lodash is already a peer dependency, using it for debounce. don't think it'll affect bundle size since other packages already rely on it